exercises/resistor-colors/README.md

```diff
@@ -0,0 +1,52 @@
+# Resistor Colors
+
+If you want to build something using a Raspberry Pi, you'll probably use _resistors_. For this exercise, you need to know two things about them:
+
+* Each resistor has a resistance value.
+* Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
+To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band acts as a digit of a number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
+
+In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take two colors as input, and output the correct number.
+
+The band colors are encoded as follows:
+
+- Black: 0
```

Comment:
Traditionally, these bands are expressed as starting with value 1.  It fits the mnemonic that we use to remember the values.  Since this describes the problem domain, there is no reason to order them as we order things in the programming domain (often using 0 as the starting point).

Reply:
That's not what I found.

For quick reference:
https://en.wikipedia.org/wiki/List_of_electronic_color_code_mnemonics
All these mnemonics starts with BB, for Black and Brown.

In Dutch it is `**Z**ij Bracht Rozen op Gerrits Graf bij view grauw weer`
Z for Zwart = Black. `weer` = Wit = White

The fun is: it's one of the rare real life cases where the index 0 _does_ have meaning, and that was the inspiration for the exercise.  :joy:

Reply:
Well, it has been quite a few decades. :)  And I am going off of my rusty memory.  Good to go then.